### PR TITLE
Feature/signin signup

### PR DESCRIPTION
**Pull request by ashwin2201**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/form-control": "^1.5.9",
```
Comment (RealDyllon) [inline package.json]: this is not needed

File: src/index.css [hunk 2]
```diff
@@ -5,6 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  min-height: 100vh;
```
Comment (RealDyllon) [inline src/index.css]: do we need this to be global?

File: package.json (hunk 1 above)
Comment (ashwin2201) [reply package.json]: I have removed it

Comment (RealDyllon) [reply package.json]: nice

File: src/index.css (hunk 2 above)
Comment (ashwin2201) [reply src/index.css]: Removed it

Comment (RealDyllon) [reply src/index.css]: great!

**Review by RealDyllon — DISMISSED** [review]
LGTM

**Review by RealDyllon — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**